float64 cmd
---
bool status
